docs/manage/partitions/run-search-against-partition.md

```diff
@@ -23,6 +23,32 @@ Data that you ingest that is not directed to a partition will go to the default
 _index=sumologic_default
 ```
 
+## Index aliasing
+
+With index aliasing, you can create an alias to point to one or more system indexes such as [`sumologic_default`](/docs/manage/partitions/run-search-against-partition/#search-the-default-partition) in the source expression of your search query. Both the operator part and results will consist of actual index names.
+
+In addition to `sumologic_default`, we have several other Sumo Logic-defined system indexes. As a shortcut, rather than prefacing `sumologic_` when referencing system indexes in a search, you can alias these indexes by typing an underscore at the beginning. For example, `sumologic_default` and `_default` will return the same results. 
+
+:::warning Leading Underscore Reserved for System Index Alias in User-Created Indexes
+When creating your own indexes (user-created, non-system indexes), you cannot lead with an underscore (`_`). This is reserved only for system indexes.
+:::
+
+If your search query scans both your own indexes and Sumo Logic indexes starting with an underscore (`_`), you'll only see your own indexes in the results. System indexes would be ignored, and you'd see an error stating: `System indexes with alias names have been excluded from the results of the query`. 
+
+### Limitations
+
+Aliasing is not permitted for Sumo Logic system security indexes ([Cloud SIEM](/docs/cse/records-signals-entities-insights/search-cse-records-in-sumo#partitions-for-cse-records), [Audit Index](/docs/manage/security/audit-indexes/audit-index/), and [Root Cause Explorer](/docs/observability/root-cause-explorer)). These indexes do not start with `sumologic_` and therefore cannot be aliased. <!-- cross-reference this in security docs --> For all other Sumo Logic-created indexes, aliasing is permitted.
+
+### Using index aliases
+
+Here are some examples where index aliasing is used in wildcard queries.
+
+`_index=*volume*` This query will reference all type of indexes (system indexes as well as user-created).
+
+`_index=_vol*`. This query will reference indexes such as `sumologic_volume`.
+
+`_index=_*`. This will show all Sumo Logic-defined system indexes would show in search results.
```

Comment:
```suggestion
`_index=_*`. This will reference all Sumo Logic-defined system indexes.
```

Reply:
Looks good, committing this. cc: @kimsauce